packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -534,6 +534,7 @@ The Authenticator object defines the following properties:
 | `created`     | String (ISO-8601)                                               | Timestamp when the Authenticator was created                                   |
 | `id`          | String                                                          | A unique identifier for the Authenticator                                                |
 | `key`         | String                                                          | A human-readable string that identifies the Authenticator                                |
+| `status`      | `ACTIVE`,`INACTIVE`                                             | Status of the Authenticator                                |
```

Comment:
Ref: https://github.com/okta/okta-core/blob/master/components/core/factors/api/src/main/java/com/saasure/db/dto/core/entity/authenticators/Authenticator.java#L90-L93

Not sure if we need to document more possible statuses here.